### ISSUE TITLE
Chord symbol properties accessibility fixes

### ISSUE THREAD

**Issue by avvvvve** [musescore/MuseScore#29992]
Things to fix:

1. Navigation order of section is wrong and some buttons are skipped completely (see video)
     - Can't focus on Literal/Jazz, the right-align button, the three-dot button for alignment, or 'Add fretboard diagram'
2. Alignment buttons should have tooltips & screen-reader titles (right now the two that you _can_ focus on are read unhelpfully as "alignment to notehead undefined")
     - Tooltips should read as follows, following the form of tooltips for text alignment buttons:
     - **Align left** / Align left edge of text to notehead
     - **Align center** / Align horizontal center of text to notehead
     - **Align right** / Align right edge of text to notehead
3. 'Alignment to notehead' should be renamed to just 'Alignment' so that it's not truncated (tooltips described above will do the job of communicating that the alignment is to the notehead)

https://github.com/user-attachments/assets/a5ad0d5c-f200-4796-ba99-56711d4286c5
